docs/index.md

```diff
@@ -936,6 +936,10 @@ Define a global variable name. For example, suppose your app deliberately expose
 
 By using this option in conjunction with `--check-leaks`, you can specify a whitelist of known global variables that you _expect_ to leak into global scope.
 
+### `--posix-exit-codes`
+
+Exits with standard POSIX exit codes instead of the number of failed tests.
```

Comment:
[Docs] Users will probably be curious about this:

```suggestion
Exits with standard POSIX exit codes instead of the number of failed tests.

Those exit codes are:

* `0`: if all tests passed
* `1`: if any test failed
* `128 + <signal>` if given a signal, such as:
  * 134: `SIGABRT` (`128 + 6`)
  * 143: `SIGTERM` (`128 + 15`)
```